spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md

```diff
@@ -78,29 +76,26 @@ spring.cloud.azure.appconfiguration.stores[0].label=[my-label1], [my-label2]
 
 Multiple labels can be separated with comma, if duplicate keys exists for multiple labels, the last label has highest priority.
 
-### Watch configuration change
+### Configuration Refresh
 
-Watch feature allows the application to load the latest property value from configuration store automatically, without restarting the application.
+Configuration Refresh feature allows the application to load the latest property value from configuration store automatically, without restarting the application.
 
-By default, the watch feature is disabled. It can be enabled with below configuration:
-
-```properties
-spring.cloud.azure.appconfiguration.watch.enabled=true
-```
-
-Change certain property key in the configuration store on Azure Portal, e.g., /application/config.message, log similar with below will be printed on the console.
+Changing a property key in the configuration store on Azure Portal, e.g., /application/config.message, log similar with below will be printed on the console.
 
 ```console
 INFO 17496 --- [TaskScheduler-1] o.s.c.e.event.RefreshEventListener       : Refresh keys changed: [config.message]
 ```
 
 The application now will be using the updated properties. By default, `@ConfigurationProperties` annotated beans will be automatically refreshed. Use `@RefreshScope` on beans which are required to be refreshed when properties are changed.
+
 By default, all the keys in a configuration store will be watched. To prevent configuration changes are picked up in the middle of an update of multiple keys, you are recommended to use the watched-key property to watch a specific key that signals the completion of your update so all configuration changes can be refreshed together.
 
 ```properties
 spring.cloud.azure.appconfiguration.stores[0].watched-key=[my-watched-key]
 ```
 
+For web applications a refresh will be attempted whenever a ServletRequestHandledEvent occurs after the cache expiration time. Otherwise, calling refreshConfiguration on `AzureCloudConfigRefresh` will result in a refresh if the cache has expired.
+
 ### Failfast
 
 Failfast feature decides whether throw RuntimeException or not when exception happens. By default, failfast is enabled, it can be disabled with below configuration:
```

Comment:
Maybe it's worth calling out "Failfast feature decides whether to throw `RuntimeException` or not when exception happens when the configuration is initially loaded from App Configuration upon application startup."?